test/ui-testing/editor.test.js

```diff
@@ -0,0 +1,59 @@
+const timeout = process.env.SLOWMO ? 60000 : 10000;
+const fs = require('fs');
+let editor;
+
+beforeAll(async () => {
+  path = fs.realpathSync('file://../examples/index.html');
+  // scriptPath = fs.realpathSync('file://../node_modules/jquery/dist/jquery.min.js');
+  // await page.addScriptTag({path: scriptPath});
+  await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+  await page.evaluate(() => {
+    editor = new PL.Editor({
+      textarea: document.querySelector('.ple-textarea')
+    });
+  });
+});
+
+describe('Errors', () => {
+  test('Editor exists and has a textarea', async () => {
+    expect( await page.evaluate(() => document.querySelector('.ple-textarea')) ).toBeDefined();
+    expect( await page.evaluateHandle(() => editor) ).toBeDefined();
+    expect( await page.evaluateHandle(() => editor.options.textarea) ).toBeDefined();
+    expect( await page.evaluateHandle(() => editor.options.textarea.innerHTML) ).toEqual(await page.evaluateHandle(() => document.querySelector('.ple-textarea').innerHTML));
```

Comment:
It looks like comparing strings worked here, re: `expect(editor.options.textarea).toBe($('.ple-textarea')[0])` from your initial comment? I think this test looks good to go!